setup.py

```diff
@@ -24,6 +25,12 @@
 VERSION = open('VERSION', 'r').read().strip()
 VIRTUAL_ENV = hasattr(sys, 'real_prefix')
 
+python_version = platform.python_version()
```

Comment:
Thanks clebergnu ~ 
And i have one question, do we need to think about the version older than 2.6 ?
From the requirements.txt, it define :
> stevedore>=1.8.0; python_version >= '2.7'
>  \#stevedore-1.11.0 won't support python2.6 anymore
> stevedore>=1.8.0,<=1.10.0; python_version < '2.7'

So how about the following modification ?
> sys.version_info[:2] >= (2, 7)
> sys.version_info[:2] < (2, 7)